config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -18,13 +18,40 @@ presubmits:
         - --config=ci
         - --nobuild_tests_only
         - //...
+        - -//hack:verify-golangci-lint
         env:
         - name: BAZEL_FETCH_PLEASE
           value: //...
     annotations:
       testgrid-dashboards: presubmits-test-infra
       testgrid-tab-name: bazel
 
+  - name: pull-test-infra-golangci-lint
+    branches:
+    - master
+    always_run: true
+    optional: true
+    decorate: true
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/launcher.gcr.io/google/bazel:v20210204-b06ec78-test-infra
+        command:
+        - hack/bazel.sh
+        args:
+        - test
+        - --config=ci
+        - --nobuild_tests_only
+        - //hack:verify-golangci-lint
```

Comment:
Maybe also put this into a distinct PR so we can check here if everything works?

Reply:
So do you mean 2 PRs:
* make changes to `pull-test-infra-bazel` and add an empty `verify-golangci-lint` test-suite
* add the optional Job and the implementation of `verify-golangci-lint`

Or to split this into 3 PRs?

Reply:
One PR to add the exclude, an empty target and the new job as optional, and then verify everything on this PR

Reply:
Got it, So my version + also add the new optional Job in the first PR (just to confirm)

Reply:
yeah, exactly